packages/network/package.json

```diff
@@ -79,8 +91,7 @@
     "nice-grpc-web": "^2.0.1",
     "observable-webworker": "^4.0.1",
     "rxjs": "^7.5.5",
-    "threads": "^1.7.0",
-    "viem": "^0.2.0"
+    "threads": "^1.7.0"
```

Comment:
not directly related to this PR but since we're changing the package.json anyway - should we move all those peer dependencies to regular dependencies?

Reply:
I'm gonna take a pass at this in a follow up: https://github.com/latticexyz/mud/issues/411